src/storage/states/client.state.ts

```diff
@@ -0,0 +1,20 @@
+export class ClientState {
+    public constructor(
+        public id: number,
+        public userId: number,
+    ) {}
+
+    public static fromMutationPayload(json: any): ClientState {
+        return new ClientState(
+            json.ID,
+            json.UserID,
+        );
+    }
+
+    public static fromJson(json: any): ClientState {
+        return new ClientState(
+            json.id,
+            json.user.id,
```

Comment:
```suggestion
            json.userId,
```

We are currently calling this when deserializing from `localStorage` and not from GraphQL